KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/RawText.java

```diff
@@ -225,30 +221,31 @@ private void initialize() {
      *                    chunks. This means we need to address the content
      *                    pages specially.
      *
-     * @return the extracted content
+     * @return the extracted text
      *
-     * @throws SolrServerException if something goes wrong
+     * @throws NoOpenCoreException If no Solr core is available.
+     * @throws SolrServerException If there's a Solr communication or parsing issue.
      */
-    private String getContentText(int currentPage, boolean hasChunks) throws SolrServerException {
+    private String getContentText(int currentPage, boolean hasChunks) throws NoOpenCoreException, SolrServerException {
         final Server solrServer = KeywordSearch.getServer();
 
         if (hasChunks == false) {
             //if no chunks, it is safe to assume there is no text content
             //because we are storing extracted text in chunks only
             //and the non-chunk stores meta-data only
-            String name = content.getName();
             String msg = null;
+            
             if (content instanceof AbstractFile) {
                 //we know it's AbstractFile, but do quick check to make sure if we index other objects in future
-                boolean isKnown = TskData.FileKnown.KNOWN.equals(((AbstractFile) content).getKnown());
+                boolean isKnown = TskData.FileKnown.KNOWN.equals(((AbstractFile)content).getKnown());
                 if (isKnown && KeywordSearchSettings.getSkipKnown()) {
-                    msg = NbBundle.getMessage(this.getClass(), "ExtractedContentViewer.getSolrContent.knownFileMsg", name);
+                    msg = NbBundle.getMessage(ExtractedContentViewer.class, "ExtractedContentViewer.getSolrContent.knownFileMsg", content.getName());
                 }
             }
-            if (msg == null) {
-                msg = NbBundle.getMessage(this.getClass(), "ExtractedContentViewer.getSolrContent.noTxtYetMsg", name);
+            if(msg == null) {
+                msg = NbBundle.getMessage(ExtractedContentViewer.class, "ExtractedContentViewer.getSolrContent.noTxtYetMsg", "file");
```

Comment:
If this code executes, i.e., it is not a known file, the "no indexed text" message (see above) can be used.